Unix/tools/allocdump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Script to run to collect memory allocation information for all running OMI processes
+# Note that in order for this to work:
+#   1.  Omi processes must have been built with debug turned on
+#   2.  Omi loglevel must be INFO or higher
+#   3.  Current user needs to be enabled for sudo
+
```

Comment:
Also add:

```
#  4. This script only supports Linux platforms
```

Then verify that you're on a Linux platform and, if you're not, abort with an error.